### PR TITLE
Add library selection to "Aux file import"

### LINKED ISSUE #10472: Add library selection to "Aux file import"
At "Tools" > "New sub library based on AUX file", JabRef generates a new .bib file based on the cited entries

image

The dialog shows then following

image

Currently, JabRef uses the currently opened library. This may lead to confusion.

We need a library selector (similar to the import dialog)

image

Hints:

For the import dialog, the implementation was done at https://github.com/JabRef/jabref/pull/9402

### PR DESCRIPTION
**Pull request by caitlinlilley**
Fixes #10472

Screen Shot 2023-10-19 at 10 22 46 pm

### Mandatory checks
- [X] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [X] Manually tested changed features in running JabRef (always required)
- [X] Screenshots added in PR description (for UI changes)
- [X] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [X] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

File: src/main/resources/build.properties [hunk 1]
```diff
@@ -6,3 +6,4 @@ springerNatureAPIKey=${springerNatureAPIKey}
 astrophysicsDataSystemAPIKey=${astrophysicsDataSystemAPIKey}
 ieeeAPIKey=${ieeeAPIKey}
 biodiversityHeritageApiKey=${biodiversityHeritageApiKey}
+Select\ library=Create property
```
Comment (Siedlerchr) [inline src/main/resources/build.properties:9]: this looks like it came accidentally here. Pleaes remove

Comment (caitlinlilley) [reply src/main/resources/build.properties]: Thank you, I have removed this line.

File: src/main/java/org/jabref/gui/auximport/FromAuxDialog.java [hunk 2]
```diff
@@ -99,4 +102,27 @@ private void browseButtonClicked() {
...
+                    }
+                    if (database.getLocation() == DatabaseLocation.SHARED) {
+                        return database.getDBMSSynchronizer().getDBName() + " [" + Localization.lang("shared") + "]";
+                    }
+
+                    if (dbOpt.isEmpty()) {
+                        return Localization.lang("untitled");
+                    }
+
+                    return dbOpt.get();
```
Comment (koppor) [inline src/main/java/org/jabref/gui/auximport/FromAuxDialog.java:124]:
```suggestion
                    return dbOpt.orElse(Localization.lang("untitled"));
```

File: src/main/java/org/jabref/gui/auximport/FromAuxDialog.java [hunk 3]
```diff
@@ -62,10 +67,8 @@ public FromAuxDialog(JabRefFrame frame) {
             }
             return null;
         });
-
         themeManager.updateFontStyle(getDialogPane().getScene());
     }
-
```
Comment (koppor) [inline src/main/java/org/jabref/gui/auximport/FromAuxDialog.java:68]: Please keep the empty line here.

**Review by koppor — CHANGES_REQUESTED** [review]
Minor code comments

**Review by koppor — DISMISSED** [review]

**Comment by Siedlerchr** [comment]
l10n seems to be missing

**Review by koppor — APPROVED** [review]

**Comment by Siedlerchr** [comment]
There were only some http error codes for checking the help pages

**Pull request merged**